model/lifecycle.go

```diff
@@ -1581,6 +1567,11 @@ func addNewTasks(ctx context.Context, batchTimeInfo batchTimeTasksAndVariants, v
 		return nil, err
 	}
 
+	taskIdTable, err := getTaskIdTables(v, p, pairs)
```

Comment:
nit -- this function is now plural but in most places we still refer to it as singular (for example, this variable, errors, NewPatchTaskIdTable). Can we choose a convention and then make it consistent?

Reply:
done